### PR TITLE
More Excelsior Disks

### PR DESCRIPTION
**Pull request by zombiemimic**
## About The Pull Request
This is just me wanting to have more variety in the one-off Excel disks than the Drozd

## Why It's Good For The Game
I noticed how only the Drozd was able to be found in disks, and thought that it was odd to only have one Excelsior gun printable by non-infiltrators. As such, I decided to make the disks myself. 

## Changelog
:cl:
add: Added 4 new Excelsior disks, in the style of the Drozd disk that was already found.
/:cl:

### CODE REVIEW

**Comment by Kurgis** [comment]
Travis failed due to an issue with master, not with this PR.

File: code/game/objects/items/weapons/design_disks/autolathe_disks.dm [hunk 1]
```diff
@@ -110,6 +110,70 @@
 		/datum/design/autolathe/ammo/msmg/rubber
 	)
 
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/guns/ex_ak47
+	disk_name = "Excelsior - .30 AK47 Rifle"
```
Comment (Firefox13) [inline code/game/objects/items/weapons/design_disks/autolathe_disks.dm:114]: It's called ak47 only in code, let's keep in that way?
```suggestion
	disk_name = "Excelsior - .30 Kalashnikov Rifle"
```

**Review by Firefox13 — APPROVED** [review]
ignore that comment that I posted about ak47 I'm very dumb

**Comment by Omni44** [comment]
I dunno. I feel that maybe if excel disks were properly rare that would be okay, but Drozd disk already spawns everywhere. I have literally not seen round without it in past few weeks, and due to no license and drozd and all excel weapons being good it's hugely craved. I think they are kind of a balance issue.

**Comment by zombiemimic** [comment]
> I dunno. I feel that maybe if excel disks were properly rare that would be okay, but Drozd disk already spawns everywhere. I have literally not seen round without it in past few weeks, and due to no license and drozd and all excel weapons being good it's hugely craved. I think they are kind of a balance issue.

How much rarer would you like them?

**Comment by Kurgis** [comment]
Travis issue should now be resolved, close and re-open PR to restart Travis check.

**Pull request merged**